### DISCUSSION TITLE
SVG Files Open with Linked .bmp as a Separate Image

### DISCUSSION THREAD

**Discussion by staufferbrian** [d2phap/ImageGlass#1627] [General]
Hi,
I have some svg files that contain .bmp files that I'm trying to view with ImageGlass but when I open them and toggle through the images in that folder they all open separately, the svg file opens without the bmp included and then the bmp opens by itself. These files open correctly in my web browser and I've seen image glass handle them on other PC's, so I was wondering if I need to change any settings to do that? All of the files are in the same folder and SVG file is using xlink to import the bmp images.

**Comment by d2phap** [comment]
You may try the [ImageGlass Moon 9.0 beta 5](https://github.com/d2phap/ImageGlass/releases/tag/9.0.5.726) which uses Webview2 for viewing SVG

**Reply by staufferbrian** [reply]
Thanks, I just tried that and am getting the same result. We have an older computer that is running 4.5.11.27 and works fine, I tried installing that same version without any luck either. I didn't see anything unique in the settings for the program on that PC either.

**Reply by d2phap** [reply]
Version 4.5 requires ".NET Framework" 4.0 (or later), you may check if it's installed

**Reply by staufferbrian** [reply]
I verified I have version 4.0 installed. I did find that the files I opened on the other computer work ok on this one, they are coming from different sources and the xml is written a little differently.

The ones that won't work call the image by
image width="5315" height="3716" href="32 230802CA 82637571 RP2.1 v1.0.bmp" /

The ones that work have "xlink:" in front of the href so I manually added that in and it works fine that way.
image width="5315" height="3716" xlink:href="32 230802CA 82637571 RP2.1 v1.0.bmp" /

Unfortunately these are coming from a 3rd party software so we can't edit how they are built, but this at least explains why there is a difference and that it's not due a difference in the software.

Thanks for your response on this.
